.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -87,7 +87,7 @@ jobs:
           make "${KATA_ASSET}-tarball"
           build_dir=$(readlink -f build)
           # store-artifact does not work with symlink
-          sudo cp -r "${build_dir}" "kata-build"
+          mkdir kata-build && cp "${build_dir}/kata-static-*.tar.xz" kata-build/.
```

Comment:
Hi @zvonkok !

Let's make CI (almost) sudo-less! \o/

`kata-build` shouldn't exist but if for some miracle it does then `mkdir` fails. It is guaranteed  `mkdir -p` will be succeeded.

Apart from that, the code fails:
```shell
$ build_dir=$(readlink -f build)
$ mkdir kata-build && cp "${build_dir}/kata-static-*.tar.xz" kata-build/.
cp: cannot stat '/home/wmoschet/src/github.com/kata-containers/kata-containers/tools/packaging/kata-deploy/local-build/build/kata-static-*.tar.xz': No such file or directory
$ ls build
agent  agent-builder-image-version  agent-sha256sum  agent-version  kata-static-agent.tar.xz
$ ls $build_dir 
agent  agent-builder-image-version  agent-sha256sum  agent-version  kata-static-agent.tar.xz
$ mkdir kata-build && cp "${build_dir}"/kata-static-*.tar.xz kata-build/.
mkdir: cannot create directory ‘kata-build’: File exists
$ mkdir -p kata-build && cp "${build_dir}"/kata-static-*.tar.xz kata-build/.
$ ls kata-build/
kata-static-agent.tar.xz
```
It works with `mkdir -p kata-build && cp "${build_dir}"/kata-static-*.tar.xz kata-build/.`  (notice the `"${build_dir}"/kata-static-*.tar.xz`)

Reply:
Ha it is not a miracle, for each asset we do this hence if we build a couple of assets (matrix.asset) and we do it will fail let me make this better good catch.